File: 0485-max-consecutive-ones/0485-max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int ans=0;
        int maxi=0;
        for(int i=0; i<nums.size(); i++){
            ans++;
            if(nums[i]==0){
                ans=0;
            }
            maxi=max(ans,maxi);



        }

        return maxi;
    }
};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        maxi=0
        count=0

        for i in range(0,len(nums)):
            if(nums[i]==1):
                count=count+1
            
            else:
                count=0
            maxi=max(count,maxi)
        
        return maxi